jira-slack-server-integration/jira-slack-server-integration-plugin/src/test/java/com/atlassian/jira/plugins/slack/util/CommentUtilTest.java

```diff
@@ -0,0 +1,42 @@
+package com.atlassian.jira.plugins.slack.util;
+
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CommentUtilTest {
+
+    @Test
+    void testRemoveJiraTagsRemovesValidTags() {
+        assertEquals("Some text with a title",
```

Comment:
Multiple similar assertions are a good candidate to "table tests" or "parameterized tests". See this documentation, for instance: https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests-sources-MethodSource

Reply:
I added parameterized tests instead of simple tests